src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/SwapGrTags.hpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "Domain/Tags.hpp"
+#include "Evolution/DgSubcell/ActiveGrid.hpp"
+#include "Evolution/DgSubcell/Tags/ActiveGrid.hpp"
+#include "Evolution/DgSubcell/Tags/Inactive.hpp"
+#include "Evolution/DgSubcell/Tags/Mesh.hpp"
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/System.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+template <size_t Dim>
+class Mesh;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+template <typename TagsList>
+class Variables;
+/// \endcond
+
+namespace grmhd::ValenciaDivClean::subcell {
+/*!
+ * \brief Swaps the inactive and active GR variables.
+ *
+ * The values on the subcells are at the cell-centers.
+ *
+ * It should be possible to reduce memory usage by deallocating the GR variables
+ * on the DG grid when switching to subcell. However, the opposite case is not
+ * true since the GR variables are needed on the subcells if a neighbor is using
+ * subcell in order to compute the neighbor's fluxes.
+ */
+struct SwapGrTags {
+  using return_tags = tmpl::list<typename System::spacetime_variables_tag,
+                                 evolution::dg::subcell::Tags::Inactive<
+                                     typename System::spacetime_variables_tag>>;
+  using argument_tags =
+      tmpl::list<::domain::Tags::Mesh<3>, evolution::dg::subcell::Tags::Mesh<3>,
+                 evolution::dg::subcell::Tags::ActiveGrid>;
+
+  static void apply(
```

Comment:
Maybe add a clarifying comment that
- `active_grid` is the grid we are swapping to
- on output, the `active_gr_vars` will match the `active_grid`, but on input it is allowed to match or not-match (phrased better? :P)